src/components/Button/Button.tsx

```diff
@@ -0,0 +1,41 @@
+import { ReactNode } from 'react';
+import { BaseStyleProp } from '../types';
+import { makeStyles } from '../../utils/makeStyles';
+import { sx } from '../../utils/sx';
+
+type ButtonStyles = 'root' | 'primary' | 'secondary';
+
+export interface ButtonProps extends BaseStyleProp<ButtonStyles> {
+  children?: ReactNode;
+  variant?: 'primary' | 'secondary';
+  href: string;
+}
+
+const useStyles = makeStyles({
+  root: {
+    fontSize: '18px',
+    textDecoration: 'none',
+    padding: '10px 16px',
+    borderRadius: '5px',
```

Comment:
let's remove this. We will add these for styled-components.

Reply:
I added these stylings here for `Button` because we have another component called `Link` which is a simple `a` tag without any styling as suggested by you. If we remove these stylings for the `Button`, then it will become the same as `Link`, nothing different.

But fine, I'm removing these styles from here.

@agentmilindu